gcp_variant_transforms/libs/bigquery_vcf_data_converter.py

```diff
@@ -56,6 +49,19 @@
     bigquery_util.ColumnKeyConstants.CALLS_PHASESET
 ]
 
+# Constants for calculating row size. This is needed because the maximum size of
+# a BigQuery row is 100MB. See
+# https://cloud.google.com/bigquery/quotas#import for details.
+# We set it to 90MB to leave some room for error as our row estimate is based
+# on sampling rather than exact byte size.
+_MAX_BIGQUERY_ROW_SIZE_BYTES = 90 * 1024 * 1024
+# Maximum number of calls to sample for BigQuery row size estimate.
+_MAX_NUM_CALL_SAMPLES = 5
```

Comment:
If I understand correctly, this is actually the min number of calls (we always sample 5 calls or 5+1 calls). Maybe `NUM_CALL_SAMPLES` is good enough.

Reply:
Good point! Done!